lib/grizzly/zipgateway/log_monitor.ex

```diff
@@ -158,7 +181,44 @@ defmodule Grizzly.ZIPGateway.LogMonitor do
     {:noreply, %{state | home_id: home_id}}
   end
 
-  def handle_info({:message, _}, state) do
-    {:noreply, state}
+  def handle_info({:message, "Serial Process init" <> _}, state) do
+    {:noreply, set_sapi_status(state, :initializing)}
+  end
+
+  def handle_info({:message, "Bridge init done" <> _}, state) do
+    {:noreply, set_sapi_status(state, :ok)}
+  end
+
+  def handle_info({:message, message}, state) do
+    cond do
+      not String.contains?(message, "SerialAPI: Retransmission") ->
+        {:noreply, state}
+
+      state.sapi_retransmissions > 4 ->
```

Comment:
Is there ever a case where we'd want this adjusted? Like some cases 4 is fine, others we want more, less, etc?

Reply:
I've never seen a case where it happened more than 4 times and then recovered on its own.